payments-core/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -86,6 +86,12 @@ open class StripeEditText @JvmOverloads constructor(
 
     private var errorMessageListener: ErrorMessageListener? = null
 
+    private val isLastKeyDeleteTextWatcher = object : StripeTextWatcher() {
```

Comment:
should we also add a test in `StripeEditTextTest`?

Reply:
Added some!